### ISSUE TITLE
Images not saving properly in posts

### ISSUE THREAD

**Issue by Nicknyr** [steemit/condenser#2820]
I'm trying to post my second ever post on Steemit and I am running into this error when I attempt to upload images:

An image in this post did not save properly.

My images all look something like this:

```
![Screenshot from 2018-05-25 20-20-47.png](https://cdn.steemitimages.com/DQmQqyx8n5t5jk1oAXssv5UqQvkR5bkwAxbgaj69Mb3Xukh/Screenshot%20from%202018-05-25%2020-20-47.png)
<br>
```

I've tried removing the break tag that comes after because I thought maybe that was causing the problem. Still doesn't work. The odd thing is I have 5 images and all 5 following the same convention but there is an error with only 3 of them.

What's going on with this?

**Comment by Nicknyr** [comment]
Tried several. Looks like this is the problem I'm running into: https://github.com/steemit/condenser/issues/2822

**Comment by Smittyvb** [comment]
The example you posed has no query parameters, and many image hosts ignore them anyways. This appears to be a different issue from #2822. This appears to be an issue when you upload images directly in the editor, as the example you posted uses steemitimages.com. Try using a host that isn’t steemitimages. Could you post your post on an interface without this posting restriction ([Busy](https://busy.org), is one), so we can see if Steemit should have prevented the post?

**Comment by Nicknyr** [comment]
Tried using multiple image hosts including imgur.com and also tried using the Steemit editor to post the images. Even when I remove all the images from my post and try to post it I still get an image error.

**Comment by Smittyvb** [comment]
It works for me (from a testing account): https://steemit.com/test/@likesmylove/2820

**Comment by biffybirdcam** [comment]
I am having an issue uploading to the steemit.com editor page as well.

Except I am receiving an error:

Image Upload Error: [object object}

Whatever that means!?

I went through a whole bunch of steps, cleared caches and checked scripts. Couldn't solve it. I was uploading images just fine only a few hours prior, then this all of a sudden.

**Comment by biffybirdcam** [comment]
So it seems to be related to uploading directly into Steemit.com Editor pane.. I just uploaded to busy.org with ease and it is displaying the thumbnail before final submission.

hmm..

**Comment by Nicknyr** [comment]
The Steemit editor, both the raw HTML and toolbar version, are having issues with images. Posted the very same post to busy.org and it worked fine.

**Comment by jnordberg** [comment]
@Nicknyr what browser are you using?

The image loads fine for me <https://cdn.steemitimages.com/DQmQqyx8n5t5jk1oAXssv5UqQvkR5bkwAxbgaj69Mb3Xukh/Screenshot%20from%202018-05-25%2020-20-47.png>

**Comment by jredbeard** [comment]
This has been resolved (along with #2822). @Nicknyr if you believe that is not the case feel free to comment back.
